src/lib/jar.js

```diff
@@ -73,6 +73,21 @@ var Jar = {
         var expires = '; expires=' + new Date(new Date().setYear(new Date().getFullYear() + 1)).toUTCString();
         var path = '; path=/';
         document.cookie = obj + expires + path;
+    },
+    getSignedValue: function (cookieName, signedValue, callback) {
```

Comment:
I also wonder if the signing stuff should move into another module since it doesn't really have to do with cookies, it just happens that cookies are signed.